Source/WebCore/dom/Range.h

```diff
@@ -52,6 +52,8 @@ class Range final : public AbstractRange, public CanMakeWeakPtr<Range> {
     bool collapsed() const final { return m_start == m_end; }
     WEBCORE_EXPORT Node* commonAncestorContainer() const;
 
+    bool isLiveRange() const final { return true; }
```

Comment:
Why are we making this public?? It's a public member of `AbstractRange` so we shouldn't have to make this public here. If any code is relying on this, it means we're calling `isLiveRange` on `Range` object, which is redundant.

Reply:
Yep, I will change it!